src/app/pages/Collectibles/components/CollectibleImageLoader.tsx

```diff
@@ -11,6 +11,6 @@ interface Props {
 
 export const CollectibleImageLoader: FC<Props> = ({ large = false, className }) => (
   <div className={clsx('w-full aspect-square flex items-center justify-center', className)}>
-    <Spinner theme="dark-gray" className={large ? 'w-10' : 'w-8'} />
+    <Loader size={large ? 'XXL' : 'L'} trackVariant="dark" className="text-secondary" />
```

Comment:
I have never seen a loader of size XXL in the new design. Please agree on this variant of size with designers.

Reply:
you can find it in collectible_full base component in figma (54x54)